db/schema.rb

```diff
@@ -73,4 +85,21 @@
   add_index "in_progress_forms", ["form_id"], name: "index_in_progress_forms_on_form_id", using: :btree
   add_index "in_progress_forms", ["user_uuid"], name: "index_in_progress_forms_on_user_uuid", using: :btree
 
+  create_table "terms_and_conditions", force: :cascade do |t|
+    t.string   "name"
+    t.string   "title"
+    t.text     "text"
+    t.string   "version"
+    t.boolean  "latest",     default: false
+    t.datetime "created_at"
+    t.datetime "updated_at"
+  end
+
+  create_table "terms_and_conditions_acceptances", force: :cascade do |t|
+    t.string   "user_uuid"
+    t.integer  "terms_and_conditions_id"
+    t.datetime "created_at"
+    t.datetime "updated_at"
+  end
```

Comment:
actually looks like using references doesn't automatically add the index for foreign_key